lib/ui/platform_dispatcher.dart

```diff
@@ -801,11 +801,36 @@ class PlatformDispatcher {
   ///
   ///  * [SchedulerBinding], the Flutter framework class which manages the
   ///    scheduling of frames.
+  ///  * [forceSyncFrame], a similar method that is used in rare cases that
+  ///    a frame must be rendered immediately.
```

Comment:
I'm concerned folks might incorrectly use `forceSyncFrame` here thinking it might improve performance. Perhaps we shouldn't advertise `forceSyncFrame` at all here?

Reply:
We should not create an API like this.

People _will_ use it incorrectly and misunderstand what it is for.

This should be named as it is in the current framework usage, something like `requestWarmupFrame`.